src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/Teams/Table/role.html.twig

```diff
@@ -4,39 +4,57 @@
 {% elseif user.intendedRole is defined %}
     {% set userRole = user.intendedRole|lower %}
 {% else %}
-    {% set userRole = '' %}
+    {% set userRole = 'member' %}
 {% endif %}
-{% set whitespaceBefore %}
-    &nbsp;
-{% endset %}
-{% set iconAfter %}
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&#xf0d7;
-{% endset %}
 {% if user is defined %}
     {% set url = path('team_change_role', {'memberId': user.id, 'newRole': 'stand-in' }) %}
 {% endif %}
 
-<label for="role-select"{% if labelClass is defined %} class={{ labelClass }}{% endif %}>{% if labelText is defined %}{{ labelText }}{% endif %}</label>
-<select id="role-select" class="teams__select"{% if url is defined %} data-url="{{ url }}"{% endif %}{% if isInvite is defined and isInvite %} disabled {% endif %}>
-    <option class="fontawesome" data-icon="fa-star" value="admin" {% if userRole == 'admin' %}selected="selected"{% endif %}>
-        &#xf005;{{ whitespaceBefore }}
-        {{ 'teams.management.table.role.admin'|trans }}
-        &nbsp;&nbsp;&nbsp;{{ iconAfter }}
-    </option>
-    <option class="fontawesome" data-icon="fa-universal-access" value="owner" {% if userRole == 'owner' %}selected="selected"{% endif %}>
-        &#xf29a;{{ whitespaceBefore }}
-        {{ 'teams.management.table.role.owner'|trans }}
-        &nbsp;&nbsp;&nbsp;{{ iconAfter }}
-    </option>
-    <option class="fontawesome" data-icon="fa-black-tie" value="manager" {% if userRole == 'manager' %}selected="selected"{% endif %}>
-        &#xf27e;{{ whitespaceBefore }}
-        {{ 'teams.management.table.role.manager'|trans }}
-        {{ iconAfter }}
-    </option>
-    <option class="fontawesome" data-icon="fa-user" value="member" {% if userRole == 'member' %}selected="selected"{% endif %}>
-        &#xf007;{{ whitespaceBefore }}
-        {{ 'teams.management.table.role.member'|trans }}
-        &nbsp;{{ iconAfter }}
-    </option>
-</select>
+<div class="no-margin-bottom" tabindex="0" {% if url is defined %} data-url="{{ url }}"{% endif %}>
```

Comment:
From the looks of it, this should likely be a UL with LI's in it, rather than a div with span's in it?

Reply:
the sub-ul would then need to be in a li